loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -227,7 +229,7 @@ module BABYLON.GLTF2 {
                 const buffers = this._gltf.buffers;
                 if (buffers && buffers[0] && !buffers[0].uri) {
                     const binaryBuffer = buffers[0];
-                    if (binaryBuffer.byteLength != data.bin.byteLength) {
+                    if (binaryBuffer.byteLength < data.bin.byteLength - 3 || binaryBuffer.byteLength > data.bin.byteLength) {

```

Comment:
just for my understanding: why -3 ?

Reply:
Bin chunk must be 4-byte aligned but the buffer length does not. See https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#glb-stored-buffer and https://github.com/KhronosGroup/glTF/issues/1026.

Reply:
ok thanks a lot!